schedule/virt_autotest/install_guest_on_slem_kvm_host.yaml

```diff
@@ -5,6 +5,10 @@ description: |
   Virtualization Guest Installation on SLE Micro KVM Host
 
 vars:
+  IPXE: 1
+  IPXE_UEFI: 1
+  USB_BOOT: 1
+  CTRL_X_TO_BOOT: 1
```

Comment:
This is a worker config item in `/etc/openqa/workers.ini`, which may differ among SUTs.

Reply:
Do you want to remove ```CTRL_X_TO_BOOT``` ?

Reply:
yes , it is already in worker salt config